core/src/muxing.rs

```diff
@@ -91,14 +91,19 @@ pub trait StreamMuxer {
     /// is ready to be polled, similar to the API of `Stream::poll()`.
     /// Only the latest task that was used to call this method may be notified.
     ///
+    /// Implementors may rely on this method being called soon after the current
+    /// task has been notified. It is thus safe to use this method to perform
```

Comment:
```suggestion
    /// task has been notified. It is thus possible to use this method to perform
```

The word "safe" means something specific in Rust.